File: app/src/main/java/ru/usedesk/sample/service/CustomSimpleNotificationsService.java
package ru.usedesk.sample.service;

import android.app.PendingIntent;
import android.content.Intent;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import ru.usedesk.chat_sdk.service.notifications.UsedeskNotificationsServiceFactory;
import ru.usedesk.chat_sdk.service.notifications.view.UsedeskSimpleNotificationsService;
import ru.usedesk.sample.ui.main.MainActivity;

public class CustomSimpleNotificationsService extends UsedeskSimpleNotificationsService {

    @Nullable
    @Override
    protected PendingIntent getContentPendingIntent() {
        Intent intent = new Intent(this, MainActivity.class);
        intent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_SINGLE_TOP);
        return PendingIntent.getActivity(this, 0, intent, 0);
    }

    @Nullable
    @Override
    protected PendingIntent getDeletePendingIntent() {
        return null;
    }

    public static class Factory extends UsedeskNotificationsServiceFactory {
        @NonNull
        @Override
        protected Class<?> getServiceClass() {
            return CustomSimpleNotificationsService.class;
        }
    }
}

File: app/src/main/java/ru/usedesk/sample/service/CustomSimpleNotificationsService.kt
package ru.usedesk.sample.service

import android.app.PendingIntent
import android.content.Intent
import android.os.Build
import ru.usedesk.chat_sdk.service.notifications.UsedeskNotificationsServiceFactory
import ru.usedesk.chat_sdk.service.notifications.view.UsedeskSimpleNotificationsService
import ru.usedesk.sample.ui.main.MainActivity

class CustomSimpleNotificationsService : UsedeskSimpleNotificationsService() {
    override fun getContentPendingIntent(): PendingIntent? {
        val intent = Intent(this, MainActivity::class.java)
        intent.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_SINGLE_TOP
        return PendingIntent.getActivity(
            this,
            0,
            intent,
            when {
                Build.VERSION.SDK_INT >= Build.VERSION_CODES.M -> PendingIntent.FLAG_IMMUTABLE
                else -> 0
            }
        )
    }

    override fun getDeletePendingIntent(): PendingIntent? = null

    class Factory : UsedeskNotificationsServiceFactory() {
        override val serviceClass: Class<*> = CustomSimpleNotificationsService::class.java
    }
}